.travis.yml

```diff
@@ -8,6 +8,9 @@ php:
   - hhvm
 
 matrix:
+  include:
+    - php: 5.4
+      env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
```

Comment:
This creates an additional test on Travis for the lowest possible versions of packages, which is good to make sure that BC is maintained.